packages/contract-tests/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@odh-dashboard/contract-tests",
+  "version": "1.0.0",
+  "description": "Zero-config contract testing for ODH Dashboard packages",
+  "main": "dist/index.js",
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "tsc -p tsconfig.standalone.json && cp src/jest.d.ts dist/",
+    "type-check": "tsc -p tsconfig.json --noEmit",
+    "test-unit": "jest --passWithNoTests",
+    "test-unit:watch": "jest --watch --passWithNoTests",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "clean": "rm -rf dist"
+  },
+  "bin": {
+    "odh-ct-start-bff": "scripts/start-mock-bff.sh",
+    "odh-ct-stop-bff": "scripts/stop-mock-bff.sh",
+    "odh-ct-run": "scripts/run-consumer-tests.sh",
+    "odh-ct-bff-consumer": "scripts/run-go-bff-consumer.sh"
+  },
+  "dependencies": {
+    "axios": "^1.6.0",
+    "jest-html-reporters": "^3.1.5",
+    "@apidevtools/json-schema-ref-parser": "^11.0.0",
+    "openapi-response-validator": "^12.1.3",
+    "ajv": "^8.12.0",
+    "ajv-formats": "^2.1.1",
+    "js-yaml": "^4.1.0"
```

Comment:
Move to devDependencies for now until we figure out how to improve our red hat builds with respect to all the deps.